### DISCUSSION TITLE
Ex. 1: Very similar query+result pairs

### DISCUSSION THREAD

**Discussion by Sejmou** [sebastian-hofstaetter/teaching#60] [Advanced IR: Exercise Q&A] [answered]
I am currently working on exercise 1. I just got the same query and result pairs twice in a row, with the only difference being that the order of the words of the query was swapped:
![image](https://user-images.githubusercontent.com/31240010/162617979-3ec6b5af-c181-4ab0-959a-3fa6ce1e4bd7.png)

The output text is exactly identical in both cases, checked it with a diffchecker tool.

Just wanted to make sure whether this is ok or should be considered a bug.

I am aware that a similar  [issue](https://github.com/sebastian-hofstaetter/teaching/discussions/59#discussion-3990791) already exists. I was not sure if I should have just created another answer there instead.

Best,
Samo

**Comment by sebastian-hofstaetter** [comment, marked as answer]
Hi,
That should be fine, because only pairs need to be unique (so both the document and query ids) it is probably the same document, although it is also possible that the text is a duplicate and the ids are different, at that scale that might happen. I wouldn't worry about it.
Best,
Sebastian
